### ISSUE TITLE
a tags overlap?!

### ISSUE THREAD

**Issue by xhevatziberi** [SaraSoueidan/circular-navigation#11]
http://tympanus.net/Tutorials/CircularNavigation/index2.html
In the link above (demo 2), when you open the navigation and click <b>a</b> tags in order it works fine,
but when you click them in reverse order the last a tag clicked "overlaps" previous a tags;
you can test them with this code:
```javascript
$('ul li a').on('click', function(){
	console.log('a span: ' + $(this).find('span').text());
});
```
Try clicking "About" then "Tutorials" and again "About"

**Comment by SaraSoueidan** [comment]
Yes there are various bugs in that menu that showed up as browsers upgraded. Unfortunately, I don't have enough time to look into it at the moment, but please feel free to submit a pull request if you are able to debug it. 

More importantly, I highly recommend against using this menu today. SVG provides a much better alternative. You can generate and use an SVG circular menu using this tool: http://sarasoueidan.com/tools/circulus
